src/User32.Desktop/User32.cs

```diff
@@ -85,6 +85,32 @@ public static partial class User32
         public delegate int WindowsHookDelegate(int nCode, IntPtr wParam, IntPtr lParam);
 
         /// <summary>
+        /// An application-defined callback (or hook) function that the system calls in response to events generated by an accessible object.
+        /// The hook function processes the event notifications as required. Clients install the hook function and request specific types
+        /// of event notifications by calling <see cref="SetWinEventHook(WindowsEventHookType, WindowsEventHookType, IntPtr, IntPtr, int, int, WindowsEventHookFlags)"/>.
+        /// </summary>
+        /// <param name="hWinEventHook">Handle to an event hook function. This value is returned by SetWinEventHook when the hook function
+        /// is installed and is specific to each instance of the hook function.</param>
+        /// <param name="event">Specifies the event that occurred. This value is one of the <see cref="WindowsEventHookType"/> constants.</param>
+        /// <param name="hwnd">Handle to the window that generates the event, or NULL if no window is associated with the event.
+        /// For example, the mouse pointer is not associated with a window.</param>
+        /// <param name="idObject">Identifies the object associated with the event. This is one of the object identifiers or a custom object ID.
+        /// <see href="https://msdn.microsoft.com/en-us/library/windows/desktop/dd373606(v=vs.85).aspx"/></param>
+        /// <param name="idChild">Identifies whether the event was triggered by an object or a child element of the object.
+        /// If this value is CHILDID_SELF, the event was triggered by the object; otherwise, this value is the child ID of the element that triggered the event.</param>
+        /// <param name="dwEventThread">Identifies the thread that generated the event, or the thread that owns the current window.</param>
+        /// <param name="dwmsEventTime">Specifies the time, in milliseconds, that the event was generated.</param>
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        public delegate void WinEventProcDelegate(
```

Comment:
Yes sadly `WindowsHookDelegate` is as it is until we move the major version or we'll break semver :(